src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/getParams.tsx

```diff
@@ -88,13 +88,17 @@ const getUtcValue = (maybe: string | string[] | undefined | null): string | unde
   return undefined;
 };
 
-interface Params {
+type Params = {
   start?: string | string[] | undefined | null;
   end?: string | string[] | undefined | null;
   period?: string | string[] | undefined | null;
   statsPeriod?: string | string[] | undefined | null;
   utc?: string | string[] | undefined | null;
   [others: string]: string | string[] | undefined | null;
+};
+
+function isEntryDefined<T>(t: T): t is {[K in keyof T]: Exclude<T[K], null | undefined>} {
```

Comment:
`T` is an array from the way `isEntryDefined` is used in this file.

Reply:
An array is an object though, this is how typescript types `Object.entries()` fwiw:

![image](https://user-images.githubusercontent.com/79684/79477198-c4c54d80-7fbe-11ea-98d8-939d6227af1f.png)